k8s/tf/70_prod_db_redis/rds-multi-az/rds-multi-az.tf

```diff
@@ -29,6 +29,7 @@ resource "aws_db_instance" "sumo_rds" {
   kms_key_id                   = aws_kms_key.key.arn
   deletion_protection          = true
   performance_insights_enabled = true
+  # apply_immediately            = true
```

Comment:
Let's uncomment this line. It's not like we will accidentally apply random changes to the RDS instance then, and even if we do, we always prefer if they happen immediately rather than in some random maintenance window.